r_basics_introduction/r_basics_introduction.md

```diff
@@ -552,14 +592,14 @@ The input that goes into a function is called an **argument**. The argument to a
 
 A function can have zero, one, or many arguments. If there is more than one argument, we use commas to separate them. We'll see examples of that later.
```

Comment:
```suggestion
A function can have zero, one, or many arguments. If there is more than one argument, we use commas to separate them.
```

We don't actually see any examples like that in this module, right? I think "later" is a remnant from the workshop when all the content was in one place.